sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/theme/AppColors.kt

```diff
@@ -0,0 +1,25 @@
+package co.nimblehq.sample.compose.ui.theme
+
+import androidx.compose.material.*
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.graphics.Color
+
+/**
+ * Extend final [Colors] class to provide more custom app colors.
+ */
```

Comment:
Could you explain this a bit more? I think it would be best to just add the sample code instead of having a comment

Reading from [this document](https://gustav-karlsson.medium.com/extending-the-jetpack-compose-material-theme-with-more-colors-e1b849390d50), I suppose you mean something like this right?

```
data class AppColors(
    val themeColors: Colors,
    val topAppBarBackground: Color = GreenCitrus
) {
    val primary: Color get() = GreenCitrus
}
```

But when I first read "Extend", I thought it was supposed to be done like this:

```
data class AppColors(
    val themeColors: Colors,
    val topAppBarBackground: Color = GreenCitrus
) : Colors()
```

Please let me know if I misunderstood 🙇

Reply:
@Tuubz I will use the word `Expand` instead of `Extend` to eliminate the confusion.

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/376/commits/bfe4c1f547c6b64ea983839027bdc70c9c38a6ad